install/kubernetes/cilium/templates/cilium-operator/clusterrole.yaml

```diff
@@ -60,6 +60,7 @@ rules:
   - services/status
   verbs:
   - update
+  - patch
```

Comment:
Unrelated to this PR, but overall I think we only allow access to the service if LB IPAM is enabled via Helm